### PR TITLE
PROD-59: Add all needed gems

### PR DESCRIPTION
**Pull request by imysheniuk**
screenshot from 2018-04-04 13-25-39

# Reviewers

- [x] @gnaizer 
- [x] @mm1krob
- [x] @vikabelous
- [x] @okohuch

# Jira task
https://ssu-jira.softserveinc.com/browse/PROD-59

# Description
Need to add all gems for application.
Need to fix pull_request_template.md 

# Checklist:

- [x] Rubocop doesn't raise any warnings and offenses
- [x] All specs pass
- [x] New features are covered with new specs
- [x] PR is approved by 2 team mates
- [x] PR is approved by 2 experts

### CODE REVIEW

**Review by shevchukm — DISMISSED** [review]

**Review by gnaizer — APPROVED** [review]

**Review by viktoriia-bilous — CHANGES_REQUESTED** [review]
Please initialize rspec for your project and remove test folder

File: pull_request_template.md [hunk 1]
```diff
@@ -11,20 +19,13 @@
...
-- [ ] I have performed a self-review of my own code
+
+
 - [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
-- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (viktoriia-bilous) [inline pull_request_template.md]: I would suggest the following checklist for your project:
- rubocop doesn't raise any warnings and offenses
- all specs pass 
- new features are covered with new specs
- PR is approved by 2 team mates
- PR is approved by 2 experts

@okohuch what do you think?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```
Comment (viktoriia-bilous) [inline spec/rails_helper.rb]: Please remove comments

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (viktoriia-bilous) [inline spec/spec_helper.rb]: Same here

**Review by viktoriia-bilous — CHANGES_REQUESTED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (okohuch) [reply pull_request_template.md]: @vikabelous I like the list you suggested.
@imysheniuk update the checklist according to Vika's comment

**Comment by okohuch** [comment]
@imysheniuk I do not see you follow your agreement on PR creation we discussed. Change the name of the PR and commit.

**Comment by imysheniuk** [comment]
@okohuch , sorry, I didn't notice that PR name, which was automatically generated, doesn't match our agreement. Will be more attentive next time :)

**Comment by viktoriia-bilous** [comment]
Please update your own PR checklist according to the changes you made to PR template file

**Comment by viktoriia-bilous** [comment]
How should I know that “Rubocop doesn't raise any warnings and offenses” is true? Until you set up CI for your project, please attach a screenshot

**Review by shevchukm — APPROVED** [review]

**Review by viktoriia-bilous — CHANGES_REQUESTED** [review]
What about database for production?

File: Gemfile [hunk 4]
```diff
@@ -4,27 +4,38 @@ git_source(:github) do |repo_name|
...
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
-  gem 'spring'
+  gem 'spring', '~> 2.0.0'
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'web-console', '>= 3.3.0'
 end
 
+group :production do
+  gem 'unicorn', '~> 5.3.0'
```
Comment (okohuch) [inline Gemfile]: I do not see any DMS you are going to use in production. Is there any reason you did not add it?

**Review by viktoriia-bilous — CHANGES_REQUESTED** [review]
Please update production section in **database.yml.example** file since you will use postgres in production

**Review by viktoriia-bilous — APPROVED** [review]

File: config/database.yml.example [hunk 5]
```diff
@@ -13,8 +13,11 @@ development:
...
 production:
-  <<: *default
+  adapter: postgresql
+  encoding: utf8
   database: ProDobro_production
-  username: ProDobro
-  password: <%= ENV['PRODOBRO_DATABASE_PASSWORD'] %>
+  pool: 4
+  username:
+  password:
```
Comment (okohuch) [inline config/database.yml.example:23]: Did you check with @Retttro before you removed username and password?

Comment (imysheniuk) [reply config/database.yml.example:23]: yes, he doesn't need it. He will use own

Comment (okohuch) [reply config/database.yml.example:23]: ok, @Retttro make sure you update it in database.yml

**Review by okohuch — APPROVED** [review]

**Pull request merged**